### ISSUE TITLE
Create a more meaningful screen reader icon

### ISSUE THREAD

**Issue by danirabbit** [elementary/quick-settings#115]
### Problem

The screen reader icon is for Orca, but this is not meaningful to most people.

### Proposal

The screen reader icon should be something that directly communicates that this is a screen reader. Not an Orca

### Prior Art (Optional)

A computer screen with sound waves:

![Image](https://github.com/user-attachments/assets/fb4a0535-0e0f-4f92-ac01-7b8e8d8ac318)

Computer screen with speech bubble:

![Image](https://github.com/user-attachments/assets/c0525c43-2ed8-4c26-be99-b0c49e825edb)

A human head with sound waves:

![Image](https://github.com/user-attachments/assets/95d28397-555c-4e4d-bb1c-19ac3db45849)

Some little speech bubbles with sound icon:

![Image](https://github.com/user-attachments/assets/f35fc492-e681-4926-b1ac-ad1c4a8207a9)

A11y icon overlaid with sound icon:

![Image](https://github.com/user-attachments/assets/9e9d3ae9-932c-4bc3-8c94-042a5c427f2c)
